ITE/2/README.adoc

```diff
@@ -169,21 +169,23 @@ are three important considerations regarding the targets metadata signed
 by the delegated "`package-and-in-toto-metadata-signer`" role:
 
 [arabic]
-. Since each package is typically produced independently of other
-packages, this means that the complete set of in-toto _link_ metadata
-for a package SHOULD be different, and therefore isolated, from that for
-another package. (The in-toto _root layout_ metadata SHOULD be the same
-for every package.) This MAY be done in a number of different ways, but
-perhaps the simplest is to use directories with unique names to isolate
-different in-toto link metadata for different packages, where the name
-MAY be the SHA-256 hash of the "`developer`" step link metadata file for
-a particular package.
+. Since each package for a project is typically produced independently
```

Comment:
I added "for a project" to help clarify that different packages are still from the same abstract supply chain.